gpt_engineer/ai.py

```diff
@@ -37,8 +39,9 @@ def next(self, messages: list[dict[str, str]], prompt=None):
 
         chat = []
         for chunk in response:
-            delta = chunk["choices"][0]["delta"]
-            msg = delta.get("content", "")
+            delta = chunk['choices'][0]['delta']
+            msg = delta.get('content', '')
```

Comment:
please revert back to double quotes.

I have seen this change back and forth, it is based on personal preference now but when we add `pre-commit` you will be forced to use doubles since we are going to run `black`

Reply:
that's fine